infra/image/shcontainer

```diff
@@ -17,6 +17,7 @@ container_create() {
 
     log info "= Creating ${name} ="
     podman create \
+           --privileged \
```

Comment:
Please have a look at PR #1302 to have --privileged as an optional argument for build.sh. start.sh will not be able to start a working container when --privileged is used on Fedora.

Reply:
Fixed.